frontend/src/scss/PrimeStyles.scss

```diff
@@ -136,6 +137,17 @@
   margin-bottom: 1em;
 }
 
+// General group of related items on a form
+.sr-section {
```

Comment:
We recently had some discussion go by about migrating things out of this file into more component-specific ones. It seems like this is intended to be shared, since it's here not there, but would it be more appropriate to leave it in a component file until it's used in more than one?

Reply:
Ha, I debated that as well. I ended up with it here but we need to rearrange our CSS anyway. There are several files that are global-ish with no rhyme or reason.